index.js

```diff
@@ -247,6 +248,21 @@ const report = {
 	}
 };
 
+const cache = {
+	cache: {
+		description: 'cache instrumentation results for improved performance',
+		type: 'boolean',
+		default: true,
+		nycCommands: nycCommands.main,
```

Comment:
```suggestion
		nycCommands: nycCommands.instrument,
```

This is what actually controls where `nyc --help` displays the options, `nycCommands: nycCommands.main` displays to `nyc --help` only, `nycCommands: nycCommands.instrument` displays to `nyc --help` and `nyc instrument --help`.